lib/ansible/modules/windows/win_msp.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, Rodric Vos <rodric@vosenterprises.eu>, and others
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# this is a windows documentation stub. Actual code lives in the .ps1
+# file of the same name.
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: win_msp
```

Comment:
What do you think about renaming the module to the `win_package_msp`?

Reply:
Ah, good point, as in the comment below, it was rework on the win_msi module. So win_msp was not a bad name back then (April 2018). I think renaming is better. This thread has mind about it possibly: https://github.com/ansible/ansible/issues/22789, see if anyone else has objections.